### DISCUSSION TITLE
是否可以选择关闭搜索或者将 Search Docs 改为 搜索，自定义呢？

### DISCUSSION THREAD

**Discussion by BytesZero** [web-infra-dev/rspress#179] [Q&A] [answered]
其实对于我的项目，我是不希望有搜索的，应为他很简单就 2、3 个页面

![image](https://github.com/web-infra-dev/rspress/assets/8764899/e941919a-8e9f-419a-a170-25a3ba63556c)

我的地址：https://flutterads.github.io/site/

**Comment by BytesZero** [comment]
@sanyuan0704  文档中也没搜索到相关配置，可否解答一下

**Reply by sanyuan0704** [reply, marked as answer]
可以通过`themeConfig.search: false`关闭 https://rspress.dev/zh/api/config/config-theme.html#search
